package.json

```diff
@@ -22,7 +22,7 @@
     "prepack": "npm run build",
     "release": "scripts/release/release.sh",
     "version": "scripts/release/update-changelog-release-date.js && scripts/release/update-ethpm-version.js",
-    "test": "npm run compile && scripts/test.sh"
+    "test": "scripts/test.sh"
```

Comment:
Why this change? The explicit compile step was there to prevent truffle from recompiling on successive runs of `npm test`. Did the truffle update change this?

Reply:
This is actually what was breaking the nightly job, because the nightly build is downloaded in `scripts/test.sh`.

Note that is still the case:

https://github.com/OpenZeppelin/openzeppelin-solidity/blob/c0f69c53b8a1cc7b9c13dfe3cd7b5d7e8c7843f6/scripts/test.sh#L65-L67

Truffle is supposed to be doing this automatically, but it doesn't seem to be working.

Reply:
Oh, so it doesn't recompile on compiler changes? Shouldn't that `if ` be in a `compile` script?

Reply:
Good point...